docs/1-getting-started/creating-a-window.md

```diff
@@ -1 +1,65 @@
 # Creating a Window
+
+Creating a window using the Windows API library (a.k.a WINAPI32) can be made by following these steps:"
+
+## 1.) Fill a window class structure (WNDCLASS)
+
+```cpp
+WNDCLASS wndClass = {};
+wndClass.hInstance = hInstance; //hInstance from wWinMain/WinMain
+wndClass.lpfnWndProc = DefWindowProc; //function used to handle messages. We're currently using the default window message handling procedure. We will create our message handler function later...
+wndClass.lpszClassName = TEXT("DirectXApplication_Tutorial"); //The TEXT() macro function is used to assure the projects compiles if UNICODE support is either enabled or disabled.
+```
+
+To note that a custom cursor, application icons and flags can be set in this structure.
+
+## 2.) Register the window class
+
+We register a window class using the RegisterClass function.
+
+```cpp
+if(RegisterClass(&wndClass) == 0) 
+{
+  MessageBox(nullptr,TEXT("Cannot register the window class!"),TEXT("Error!"),MB_ICONERROR | MB_OK);
+  return -1;
+}
+```
+
+Is it most of the time safe to directly call RegisterClass, without checking if the function failed.
```

Comment:
Grammatically incorrect. Just a thought, but I feel like there could be a bit more text on this point, explaining why not checking should be fine most of the time.
> Most of the time it is safe to call RegisterClass without checking if the function failed.